docs/reference.md

```diff
@@ -229,7 +229,7 @@ func.func @check_tolerance() {
 ```
 
 This is just the first step in testing the numerical accuracy of StableHLO ops.
-At the moment, this is an underspecified area of the StableHLO spec, and there is
+At the moment, this is an under-specified area of the StableHLO spec, and there is
```

Comment:
I don't think this is the right spelling? https://en.wiktionary.org/wiki/underspecify (Same for underdocumented).

Reply:
fixed in all files